tests/store/test_footprints.py

```diff
@@ -121,34 +126,25 @@ def test_read_footprint_standard(keyword, value):
         assert footprint_data.attrs[key] == expected_attrs[key]
 
 
-def test_read_footprint_high_spatial_resolution():
+def test_read_footprint_high_spatial_resolution(standardise):
     """
     Test high spatial resolution footprint
      - expects additional parameters for `fp_low` and `fp_high`
      - expects additional coordinates for `lat_high`, `lon_high`
      - expects keyword attributes to be set
        - "spatial_resolution": "high_spatial_resolution"
     """
-    datapath = get_footprint_datapath("footprint_test.nc")
-    # model_params = {"simulation_params": "123"}
-
     site = "TMB"
-    network = "LGHG"
-    inlet = "10m"
     domain = "EUROPE"
-    model = "test_model"
-
-    bucket = get_bucket()
-    with Footprints(bucket=bucket) as fps:
-        fps.read_file(
-            filepath=datapath,
-            site=site,
-            model=model,
-            network=network,
-            inlet=inlet,
-            domain=domain,
-            period="monthly",
-            high_spatial_resolution=True,
+    standardise(
```

Comment:
Small comment - any reason why this is laid out differently than the `test_read_footprint_standard`? I like how that test was laid out, so a small suggestion would be to match this to that to keep consistent.

Reply:
Partially because I could pass keyword=value through **kwargs, but not directly. I changed it so that only keyword=value is in kwargs.

Reply:
This isn't a strong suggestion so if you feel like there's a good reason to lay it out how you did that's fine.